fe1-web/src/features/lao/reducer/LaoReducer.ts

```diff
@@ -246,6 +246,17 @@ const laosSlice = createSlice({
         }
       },
     },
+    reconnectLao: {
+      prepare: (laoId: Hash) => ({
+        payload: { laoId: laoId.valueOf() },
+      }),
+      reducer: (state, action: PayloadAction<{ laoId: string }>) => {
+        const { laoId } = action.payload;
+        if (laoId in state.byId) {
+          state.connected = true;
+        }
+      },
+    },
```

Comment:
Maybe throw an error when this is called if there is no current lao id because we don't want to have `connected=true` but `currentId != laoId`?

Reply:
Just thought about this again and what would you think about removing the laoId parameter and only check if `state.currentId` is set? Even with the changes to your PR you don't check `currentId` as far as I see?